TF_Settings_Web/src/TouchFree/Plugins/HandDataManager.ts

```diff
@@ -0,0 +1,41 @@
+import { HandFrame } from "TouchFree/TouchFreeToolingTypes";
+import { InputActionPlugin } from "./InputActionPlugin";
+
+// Class: HandDataManager
+// The manager for all <HandFrame> to be handled and distributed. This distributes the data
+// via the <TransmitHandData> event which should be listened to by any class hoping to make
+// use of incoming <HandFrame>s.
+export class HandDataManager extends EventTarget {
+    // Event: TransmitHandData
+    // An event for transmitting <HandFrame>s that are received via the <messageReceiver> to
+    // be listened to.
```

Comment:
Should this comment be here? Confused as to what Event it is describing

Reply:
This event is triggered lower down, it's how we document the events as they don't really exist like a property within JS